auslib/test/admin/views/test_rules.py

```diff
@@ -209,17 +209,11 @@ def testValidationEmptyInput(self):
         self.assertEquals(r[0]['buildID'], None)
         self.assertEquals(r[0]['version'], None)
 
-    def testInvalidMapping(self):
-        data_dict = dict(backgroundRate=31, mapping='random', priority=33, product='a', update_type='minor')
-        ret = self._post('/rules', data=data_dict)
-        self.assertEquals(ret.status_code, 400, "Status Code: %d, Data: %s" % (ret.status_code, ret.data))
-        self.assertIn("mapping", ret.data, "Status Code: %d, Data: %s" % (ret.status_code, ret.data))
-
-    def testPutDataVersionLessThanOne(self):
-        # Throw 400 error when data_version is less than 1.
-        ret = self._put('/rules/1', data=dict(backgroundRate=71, mapping='d',
-                                              priority=73, data_version=0,
-                                              product='Firefox', channel='nightly', update_type='minor'))
+    def testDuplicateAlias(self):
+        ret = self._post('/rules', data=dict(backgroundRate=31, mapping='c', priority=33,
+                                             product='Firefox', update_type='minor', channel='nightly', alias='test'))
```

Comment:
It would be good to add an assertEquals here to make sure that these first _post() is succeeding.